src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -271,3 +273,26 @@ follows. Note that if value is not one of `0`, `1`, `2` or `3`, then set to `0`.
 | 1 | 0 | 0 | |
 | 2 | GTFS `min_transfer_time` | GTFS `min_transfer_time` | Log a warning message if the `min_transfer_time` is empty |
 | 3 | 86400 | 86400 | |
+
+### Reading shapes.txt
+
+| NTFS file | NTFS field | Constraint | GTFS file | GTFS field | Note |
+| --- | --- | --- | --- | --- | --- |
+| geometries.txt | geometry__id | ID | shapes.txt | shape_id | All slashes `/` are removed |
+| geometries.txt | geometry__wkt | Required | shapes.txt | shape_pt_lat, shape_pt_lon, shape_pt_sequence | A WKT LINESTRING geometry is created from the 3 input fields. |
+
+### Reading frequencies.txt
+Frequencies are transformed into explicit passing times by creating new trips that operate on regular times within the specified period. For each line of the GTFS frequencies.txt file, the referenced trip and its stop_times are used as a sample to create the new trips whose stop_times are calculated based on the given headway.
+
+A new trip is created, departing from the first stop every `headway_secs` seconds within the time period between `start_time` and `end_time`. Stop times of the referenced trip are used to calculate the time interval between two stop departures.
+The departure time at the first stop of the last trip should not be later than the `end_time` value. In case both values for `start_time` and `end_time` are equal, the frequency is ignored (no new trip is created).
```

Comment:
equals? (not sure)

Reply:
What about adding the ignored case when endtime is inferior to the starttime ?